book/src/contributing.md

```diff
@@ -6,6 +6,64 @@ Want to help? Great! This is a fun project to hack on because it's relatively si
 
 We follow the [Rust code of conduct](https://www.rust-lang.org/policies/code-of-conduct). Please report any concerns to a [Lead][leads].
 
+## Optopodi Collaboration Guidelines
+
+Adopted from the [Mentored Sprints](https://www.mentored-sprints.dev) discord server and [The Recurse Social Rules](https://www.recurse.com/social-rules) these guidelines extend the [Rust CoC](https://www.rust-lang.org/policies/code-of-conduct) in ways that are actionable and clear examples to follow for the Optopodi team.
+```
```

Comment:
I think I would rather have this be a "bullet list" in markdown than a code block